website/docs/manage_groups.md

```diff
@@ -51,4 +60,59 @@ More levels of folder nesting are not supported.
 │   └─Japanese   <- Group
 |       └─DictB  <- Dict Files's container folder
 |          └─ DictB Files  
-```
\ No newline at end of file
+```
+
+## group by metadata.txt
+
+Click the "group by metadata" will group your dicts based on metadata.txt file.
```

Comment:
`metadata.toml`? rust's cargo.toml & python's pyproject.toml are popular, some editors may just includes a highlighter for toml